test/test_testing.py

```diff
@@ -0,0 +1,118 @@
+import os
+import subprocess
+import textwrap
+
+import pytest
+
+from . import utils
+from .template_projects import CTemplateProject
+
+project_with_a_test = CTemplateProject(
+    setup_cfg_add=textwrap.dedent(r'''
+        [options.extras_require]
+        test = nose
+    ''')
+)
+
+project_with_a_test.files['test/spam_test.py'] = r'''
```

Comment:
This, on the other hand, gets quite long, though, and won't have syntax highlighting. But I guess that's the price we have to pay to have all tests in a single file :-/

Reply:
I think it's worth it overall, but yeah, in this case it's kind of a bummer.